### PR TITLE
[preview] fix(gallery): multiple vids + wix files

### PR DESCRIPTION
**Pull request by Barackos**

### CODE REVIEW

File: packages/plugin-gallery/web/src/lib/baseUrlConverter.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const WIX_MEDIA_BASE_URL = 'https://static.wixstatic.com/';
+const WIX_VIDEO_BASE_URL = 'https://video.wixstatic.com/';
+
+export const getWixFilename = url => url.replace(WIX_MEDIA_BASE_URL, '');
+
+export const isAbsoluteUrl = url => url.startsWith('http://') || url.startsWith('https://');
+
+export const getImageAbsoluteUrl = imageUrl =>
+  !isAbsoluteUrl(imageUrl) && !imageUrl.startsWith(WIX_MEDIA_BASE_URL)
```
Comment (oded-soffrin) [inline packages/plugin-gallery/web/src/lib/baseUrlConverter.js:9]: I think isAbsoluteUrl is enough, right? both for here and for video fnc

File: packages/plugin-gallery/web/src/lib/resize-media-url.js [hunk 2]
```diff
@@ -91,14 +86,16 @@ const getResizedImageUrl = (
...
+    ? resizer(
+        getWixFilename(originalUrl),
+        item.maxWidth,
+        item.maxHeight,
+        requiredWidth,
+        requiredHeight,
+        options
+      )
+    : originalUrl;
   return retUrl;
```
Comment (oded-soffrin) [inline packages/plugin-gallery/web/src/lib/resize-media-url.js:99]: style suggestion:
if (isAbsoluteUrl(originalUrl))
    return originalUrl;

return resizer(...)

File: packages/plugin-gallery/web/src/lib/convert-item-data.js [hunk 3]
```diff
@@ -38,6 +39,14 @@ export const convertItemData = ({ items, anchorTarget, relValue }) =>
           rel: 'noopener',
           url: normalizeUrl(item.url || ''),
         };
+        const {
+          pathname,
+          thumbnail: { pathname: thumbPathname } = {},
+        } = convertedData.metaData.poster;
+        if (pathname && thumbPathname) {
+          convertedData.metaData.poster.pathname = getVideoAbsoluteUrl(pathname);
+          convertedData.metaData.poster = getImageAbsoluteUrl(thumbPathname);
```
Comment (oded-soffrin) [inline packages/plugin-gallery/web/src/lib/convert-item-data.js:48]: line 48 overrides line 47 setter (?)

Comment (Barackos) [reply packages/plugin-gallery/web/src/lib/convert-item-data.js:48]: oops!

Comment (Barackos) [reply packages/plugin-gallery/web/src/lib/convert-item-data.js]: line 47 is unnecessary, deleted

**Pull request merged**